forecast_blend/blend.py

```diff
@@ -124,7 +118,7 @@ def get_blend_forecast_values_latest(
 def add_properties_to_forecast_values(
```

Comment:
rename `properties` to `plevels`

Reply:
TODO, add in nowcasting_datamodel a column called plevels in forecast_value table

Reply:
https://github.com/openclimatefix/uk-pv-forecast-blend/pull/16